src/localize/languages/en.json

```
@@ -28,6 +30,12 @@
   "group_monitor_connected_waiting_telegrams": "Group monitor is connected and waiting for telegrams",
   "group_monitor_value": "Value",
   "group_monitor_waiting_to_connect": "Waiting for websocket connection",
+  "project_explore_title": "ETS Project",
+  "project_explore_tree_view_title": "Group Addresses",
+  "project_explore_upload": "To use the project explorer please upload a KNX project on Info tab first!",
+  "project_explore_version_l1": "The XKNXProject version used during upload and parsing of the ETS project was:",
```

```suggestion
  "project_explore_version_l1": "The `XKNXProject` version used for parsing of the ETS project was:",
```
I'd like to avoid the word "upload" since users might think we send it to some cloud or something like that.